substrate/frame/child-bounties/src/lib.rs

```diff
@@ -206,9 +220,25 @@ pub mod pallet {
 	>;
 
 	/// The description of each child-bounty.
+	///
+	/// This item replaces the `ChildBountyDescriptions` storage item from the V0 storage version.
 	#[pallet::storage]
-	pub type ChildBountyDescriptions<T: Config> =
-		StorageMap<_, Twox64Concat, BountyIndex, BoundedVec<u8, T::MaximumReasonLength>>;
+	pub type ChildBountyDescriptionsV1<T: Config> = StorageDoubleMap<
```

Comment:
I see this a bit as an experiment. Lets see if this actually helps block explorers, since we never did it like this.  
Normally we just replace the definition 🙈